src/main/java/Goods.java

```diff
@@ -0,0 +1,19 @@
+public class Goods {
```

Comment:
Можно поменять имя класса на более точное
```suggestion
public class Good {
```